docs/guide/piece-pages.md

```diff
@@ -1,257 +1,162 @@
----
-title: "Piece Pages"
----
+# Piece index and show pages
 
-# Piece Pages
+**Piece index pages** allow editors to add pages listing pieces of a particular type. They also allow pieces to have their own web pages -- known as **show pages** in Apostrophe. Show pages are automatically available once an index page and pieces of the matching piece type are created.
 
-The most intuitive example is a blog: if the individual [piece](pieces.md) is one blog post, then the piece page is the blog's home page, where all of the posts can be discovered, paginated, filtered and explored. In addition,  "piece pages" are responsible for serving the individual webpage for each piece.
+Index pages are added similarly to other [pages](/guide/pages.md) with a few important differences. Basically, these page modules have **1)** two separate template files and **2)** additional piece data available in templates. Piece page modules extend `@apostrophecms/piece-page-type`.
 
-If the project has a piece module called `product`, then that module provides a way to create, edit, manage and query pieces. If the project has a piece page module called `product-page`, then that module provides a way to browse and view the pieces.
+## Creating a piece page module
 
-In general piece modules are concerned with *editing and APIs*, while piece page modules are concerned with *browsing as part of a website*. For developers familiar with the "model / view / controller" pattern: **piece modules are the model layer for your content type, while piece page modules are the view layer.**
+A common use for these is a **blog**. The related piece type in that case might come from the `article` piece module (refer to [the pieces guide](/guide/pieces.md#creating-a-piece-type) for more on that). **The piece page would then be a module named `article-page`.**
```

Comment:
I've reread this a few times and there are a couple of things that keep confusing me:

1. We call something a blog, and then we never use the word blog again in the code. This just feels a bit weird to me Why is the blog called 'article-page'...I get it in other contexts (Event page, for example). Maybe we can avoid using blog here just to make it easier.
2. I think "related piece type" is a confusing way to say it. When looking through the Statamic examples, it feels like there is a more natural connection.